I/ITK/build_tarballs.jl

```diff
@@ -0,0 +1,125 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ITK"
+version = v"5.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/InsightSoftwareConsortium/ITK.git", "1fc47c7bec4ee133318c1892b7b745763a17d411")
+]
+#last build failed on bildkite adding comment to build again
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ITK*
+mkdir build/
+cmake -B build -S . \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DITK_USE_SYSTEM_EXPAT:BOOL=ON \
+    -DITK_USE_SYSTEM_FFTW:BOOL=ON \
+    -DITK_USE_SYSTEM_HDF5:BOOL=ON \
+    -DITK_USE_SYSTEM_JPEG:BOOL=ON \
+    -DITK_USE_SYSTEM_TIFF:BOOL=ON \
+    -DITK_USE_SYSTEM_PNG:BOOL=ON \
+    -DITK_USE_SYSTEM_EIGEN:BOOL=ON \
+    -DITK_USE_SYSTEM_ZLIB:BOOL=ON \
+    -DQNANHIBIT_VALUE:BOOL=0 \
+    -DQNANHIBIT_VALUE__TRYRUN_OUTPUT:STRING=0 \
+    -DVXL_HAS_SSE2_HARDWARE_SUPPORT:STRING=1 \
+    -DVCL_HAS_LFS:STRING=1 \
+    -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS:STRING=1 \
+    -DHAVE_CLOCK_GETTIME_RUN:STRING=0
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license ${WORKSPACE}/srcdir/ITK/LICENSE
+"""
+
+
+
+function is_problematic_platform(platform)
+    # Extract relevant information from the platform object
+    platform_string = string(platform)
+    # Check conditions to exclude the platform
+    return contains(platform_string, "i686") || #sse2 disabled errors in ITK with open issues on github
+           contains(platform_string, "macOS") || #CMAKE errors for _libcxx_run_result in cross compilation
+           contains(platform_string, "FreeBSD") || #CMAKE error for _libcxx_run_result in cross compilation
+           platform_string == "Linux x86_64 {libc=musl}" #CMAKE error for _libcxx_run_result result in cross compilation
```

Comment:
No, please don't do this by string comparison, that's too fragile. 

What was wrong with the code I suggested? It works great in other recipes.

The platform objects have methods to check cleanly for all these properties, like the OS, the bits (33 vs 64), the arch etc